assets/js/components/wp-dashboard/WPDashboardUniqueVisitorsChartWidget.js

```diff
@@ -165,9 +167,9 @@ WPDashboardUniqueVisitorsChartWidget.chartOptions = {
 		startup: true,
 	},
 	chart: {
-		title: 'Unique Visitors from Search',
+		title: 'Unique visitors over the last 28 days',
 	},
-	statsColor: '#5c9271',
+	statsColor: '#6380b8',
```

Comment:
Hey @sashadoes, sorry for not spotting this previously, but I have noticed that `statsColor` is not a standard chart option, so we should not include it here in `WPDashboardUniqueVisitorsChartWidget.chartOptions`, as the whole thing ultimately gets passed through to the Google chart.

Instead, looking at where it's used, it's evident that the `series` option is a static value, so I would suggest including the whole of the `series` option here instead of `statsColor`.

```suggestion
	series: {
		0: {
			color: '#6380b8',
			targetAxisIndex: 0,
		},
		1: {
			color: '#6380b8',
			targetAxisIndex: 0,
			lineDashStyle: [ 3, 3 ],
			lineWidth: 1,
		},
	},
```